code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -145,7 +145,7 @@
 	icon_state = "sord"
 	item_state = "sord"
 	slot_flags = SLOT_BELT
-	force = 2
+	force = 4.13
```

Comment:
4/13/2009 is the start date of homestuck. It's a minor reference tacked onto a homestuck bigger reference.